### PR TITLE
Persist oreg_url in node image

### PR DESCRIPTION
**Pull request by kargakis**
@openshift/sig-azure 

/hold 
/cherrypick release-3.10

### CODE REVIEW

**Comment by kargakis** [comment]
Testing in https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_extended_conformance_azure_39_test/1/console in conjuction with https://github.com/Azure/acs-engine/pull/3226

File: playbooks/azure/openshift-cluster/build_node_image.yml [hunk 1]
```diff
@@ -55,6 +55,12 @@
 - name: install openshift
   import_playbook: ../../openshift-node/private/image_prep.yml
 
+- name: persist oreg_url
+  copy:
+    dest: "/root/oreg_url"
```
Comment (jim-minter) [inline playbooks/azure/openshift-cluster/build_node_image.yml]: /etc ?

Comment (kargakis) [reply playbooks/azure/openshift-cluster/build_node_image.yml]: Maybe? I guess I can grab the file in acs-engine before untarring the config bundle...

Comment (kwoodson) [reply playbooks/azure/openshift-cluster/build_node_image.yml]: Oreg URL gets laid down in a few places already.  Check `/root/.docker/config.json`.  Also check `/etc/ansible/facts.d/`.  Ansible generally persists facts here.  I believe oreg_url should be there.  

If this is not found, then I might suggest a known location:

1. `/etc/origin/oreg_url`
2. `/etc/origin/config/oreg_url`

**Comment by kargakis** [comment]
Got a clean run yesterday [here](https://ci.openshift.redhat.com/jenkins/view/All/job/test_pull_request_origin_extended_conformance_azure_39_test/6/consoleFull)

This should be ready to merge I think @kwoodson @jim-minter

**Pull request merged**